Dockerfile.ocp

```
@@ -21,8 +21,6 @@ COPY --from=builder /go/src/github.com/openshift/assisted-installer-agent/build/
 COPY --from=builder /go/src/github.com/openshift/assisted-installer-agent/build/domain_resolution /usr/bin/domain_resolution
 COPY --from=builder /go/src/github.com/openshift/assisted-installer-agent/scripts/installer/* /usr/local/bin/
 
-RUN INSTALL_PKGS="biosdevname dhclient dmidecode file findutils fio ipmitool iputils nmap openssh-clients podman chrony smartmontools" && \
-    dnf install -y $INSTALL_PKGS && \
-    dnf clean all
+RUN if [[ "$(arch)" == "x86_64" ]]; then INSTALL_PKGS="biosdevname dhclient dmidecode file findutils fio ipmitool iputils nmap openssh-clients podman chrony smartmontools" && dnf install -y $INSTALL_PKGS && dnf clean all; else INSTALL_PKGS="dhclient dmidecode file findutils fio ipmitool iputils nmap openssh-clients podman chrony smartmontools" && dnf install -y $INSTALL_PKGS && dnf clean all; fi
```

just to make it easier on the eye:
```suggestion
RUN if [[ "$(arch)" == "x86_64" ]]; then \
    INSTALL_PKGS="dhclient dmidecode file findutils fio ipmitool iputils nmap openssh-clients podman chrony smartmontools biosdevname"; else \
    INSTALL_PKGS="dhclient dmidecode file findutils fio ipmitool iputils nmap openssh-clients podman chrony smartmontools"; fi \
    && dnf install -y $INSTALL_PKGS && dnf clean all
```

---

not nicer just to install install all packages that are the same in one line and special x86 one in another one?

---

Also an option, while not supporting providing two non-inclusive lists
Anything that is not just one long line of code

---

Done